docs/src/pages/guides/testing.md

```diff
@@ -36,9 +36,11 @@ const wrapper = ({ children }) => (
   </QueryClientProvider>
 );
 
-const { result } = renderHook(() => useCustomHook(), { wrapper });
+const { result, waitFor } = renderHook(() => useCustomHook(), { wrapper });
 
-expect(result.current).toEqual('Hello');
+await waitFor(() => result.current.isSuccess);
+
+expect(result.current.data).toEqual("Hello");
```

Comment:
Shouldn't `useCustomHook` return the full result object for this to work?

Reply:
Yes definitely! Sorry, must have been late...